Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -281,7 +281,7 @@ class BrowserViewController: UIViewController {
         })
 
 
-        searchLoader = SearchLoader(history: profile.history, urlBar: urlBar)
+        searchLoader = SearchLoader(profile: profile, urlBar: urlBar)
```

Comment:
Was the issue that the history property references the lazy places property which in turn results in the DB creation when first referenced? I'm starting to worry about our use of lazy properties (this is probably my fault as I started to use this pattern). I feel like we should do an audit at some point of where lazy property initialization result in side effects because this isn't the first time I've seen bugs come up from this sort of thing.